src/libutil/filter.cpp

```diff
@@ -289,8 +281,20 @@ class FilterBlackmanHarris1D : public Filter1D {
         const float A2 =  0.14128f;
         const float A3 = -0.01168f;
         const float m_pi = float (M_PI);
+#if 0
```

Comment:
I'll add a note in the comments to clarify this.
I benchmarked on both Linux and Mac and found fast_sin to be no faster than std::sin. FWIW.